platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WireBuilder.java

```diff
@@ -77,6 +89,6 @@ public Wire<T> build() {
         if (concurrent) {
             return new ConcurrentWire<>(executor, consumer);
         }
-        return new SequentialWire<>(executor, consumer);
+        return new SequentialWire<>(consumer);
```

Comment:
There should be no conceptual difference between `SequentialWire` and `ConcurrentWire`. I thought either both should assume there are executed in a ForkJoinPool, which is my preference, or both accept `Executor`.